docs/source/notebooks/SMC2_gaussians.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>"The PyMC3 model": remove this and entitle the section with this sentence just after the theoretical explanantion? This delineate the theory and the practice.</li><li> "Note that we are making two gaussians, where one has <code>w1</code> (90%) of the mass": <code>w1 = 0.1</code> above</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>